### ISSUE TITLE
Add View options to the viewport for orthographic views.

### ISSUE THREAD

**Issue by mwestphal** [f3d-app/f3d#896]
from @Da3n0n

**Comment by mwestphal** [comment]
@Da3n0n so you mean something like that: 

![image](https://github.com/f3d-app/f3d/assets/3129530/eebf78e6-1456-4b9c-b115-a7c120c1d8f9)


That feels a bit out of scope of F3D to me. wdyt @Meakk ?

**Comment by Meakk** [comment]
Isn't it related to https://github.com/f3d-app/f3d/pull/863?
Having an orthographic camera option will be implemented by #17

**Comment by mwestphal** [comment]
> Isn't it related to #863?

In a way, it is

**Comment by Meakk** [comment]
Ok, then let's close this issue once #863 is merged

**Comment by snoyer** [comment]
#886 discusses possibly more general cases and different UI options

**Comment by lknknm** [comment]
> #886 discusses possibly more general cases and different UI options
Also, see #885. My suggestion is that `5` could be the Hotkey for Ortographic view in a general way.
